hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/UserAgentSpanEnricher.java

```diff
@@ -68,6 +70,12 @@ public void enrichEvent(StructuredTrace trace, Event event) {
         return Optional.of(event.getHttp().getRequest().getUserAgent());
       }
     }
+    if (event.getGrpc() != null && event.getGrpc().getRequest() != null) {
```

Comment:
Same here, calling the attributes all the time makes it hard to read.

Reply:
Done/